<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>774</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Debug">
      <attribute name="title">
       <string>Debug</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QTextEdit" name="textEdit_receive">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>450</width>
             <height>320</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1000</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>网络设置</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <property name="horizontalSpacing">
             <number>4</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>(1)协议类型</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QComboBox" name="comboBox_ip"/>
            </item>
            <item row="9" column="0">
             <widget class="QCheckBox" name="checkBox_HexRecevive">
              <property name="text">
               <string>16进制显示</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QComboBox" name="comboBox_type">
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QPushButton" name="pushButton_CleanRecevice">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>清除接收区</string>
              </property>
             </widget>
            </item>
            <item row="13" column="0">
             <widget class="QLabel" name="LED_1">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_port">
              <property name="text">
               <string>(3)本地主机端口</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_ip">
              <property name="text">
               <string>(2)本地主机地址</string>
              </property>
             </widget>
            </item>
            <item row="14" column="0">
             <widget class="QLabel" name="LED_2">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="pushButton_Open">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>打开</string>
              </property>
             </widget>
            </item>
            <item row="12" column="0">
             <widget class="QLabel" name="LED_0">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QCheckBox" name="checkBox_time">
              <property name="text">
               <string>时间戳</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLineEdit" name="lineEdit_port"/>
            </item>
            <item row="11" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget_client" native="true">
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_ClientClose">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>断开</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_ClientIp">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_ipname">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>客户端</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>300</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="3">
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="1" column="1">
            <widget class="QPushButton" name="btn_send_clean">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>清除发送区</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btn_send">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>发送</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="2">
            <widget class="QTextEdit" name="textEdit_Send"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="1" column="2" colspan="2">
            <widget class="QCheckBox" name="checkBox_SendEnd">
             <property name="text">
              <string>发送新行</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox_TimeSend">
             <property name="text">
              <string>定时发送</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>ms</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBox_HexSend">
             <property name="text">
              <string>16进制发送</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="lineEdit_IntervalTime"/>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>周期</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="2">
            <widget class="QLabel" name="label_SendNum">
             <property name="text">
              <string>发送:0</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_ReceviceNum">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>接收:0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="Line" name="line_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1" rowspan="4">
            <widget class="Line" name="line_5">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="btn_clear_cnt">
             <property name="text">
              <string>clear</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>120</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QPushButton" name="btn_video_send_off">
             <property name="text">
              <string>video_send_off</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btn_video_send_on">
             <property name="text">
              <string>video_send_on</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="btn_pic_send_on">
             <property name="text">
              <string>pic_send_on</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Show">
      <attribute name="title">
       <string>Show</string>
      </attribute>
      <widget class="QLabel" name="picshow">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>240</y>
         <width>54</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
